### PR TITLE
use regex for check if job can be displayable

### PR DESCRIPTION
**Pull request by oziks**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,6 +32,7 @@ Please configure a new `app/scripts/config.js` file from [`app/scripts/config.js
 | DEFAULT_JOBS_PER_LINE | Default number of jobs per line on a screen |
 | REFRESH_TIME | Refresh time (ms) |
 | BACKGROUND_BLANK_SCREEN | Displays background if no job is displayed |
+| JOBS_NOT_DISPLAYED | Not displayed jobs whose title is in the RegExp |
```
Comment (KuiKui) [inline README.md]: Can you add `_REGEXP` at the end of the variable ?

File: app/scripts/config.js.dist [hunk 2]
```diff
@@ -10,6 +10,7 @@ angular.module('config', [])
         },
         'DEFAULT_JOBS_PER_LINE': 4,
         'REFRESH_TIME': 10000,
-        'BACKGROUND_BLANK_SCREEN': 'image-url-or-null'
+        'BACKGROUND_BLANK_SCREEN': 'image-url-or-null',
+        'JOBS_NOT_DISPLAYED': 'dev'
```
Comment (KuiKui) [inline app/scripts/config.js.dist]: Can you set a more explicit example ?

**Comment by KuiKui** [comment]
:+1:

File: app/scripts/e2e-mocks.js [hunk 3]
```diff
@@ -17,6 +17,16 @@ angular.module('e2e-mocks', ['ngMockE2E'])
         $httpBackend.whenGET('{#JENKINS_URL#}/view/none/api/json').respond(
             {'description':null,'jobs':[],'name':'none','property':[],'url':'https://jenkins-master.m6web.fr/view/none/'}
         );
+
+        $httpBackend.whenGET('{#JENKINS_URL#}/view/regexp/api/json').respond(function () {
+            var jobs = [
+                {'name':'regexp','url':'https://jenkins-master.m6web.fr/job/regexp/','color':'red'},
```
Comment (kennydee) [inline app/scripts/e2e-mocks.js]: Change the url please ;-)

**Pull request merged**